README.md

```diff
@@ -288,12 +288,20 @@ Once you're done, add your parser to the [zones.json](https://github.com/tmrowco
 For more info, check out the [example parser](https://github.com/tmrowco/electricitymap/tree/master/parsers/example.py) or browse existing [parsers](https://github.com/tmrowco/electricitymap/tree/master/parsers).
 
 ### Testing parsers locally
-We've added a testing server locally. In order to test your parser, you can run
+We've added a testing server locally. In order to test your parser, make sure first that you have installed the required modules as described in parsers/requirements.txt: for that you can run
+```
+pip install -r parsers/requirements.txt
+```
+
+Then, run the following command to test your parser locally:
 ```
 PYTHONPATH=. python mockserver/update_state.py <zone_name>
 ```
+
 from the root directory, replacing `<zone_name>` by the zone identifier of the parser you want to test. This will fetch production and exchanges and assign it a random carbon intensity value. It should appear on the map as you refresh your local browser.
```

Comment:
This doesn't read well (as you now have a ":" before the command.
I suggest replacing the sentence "Then, run the following command to test your parser locally:" by "Then, you can run" in the sentence just before the command.